allennlp/tests/models/bert_for_classification_test.py

```diff
@@ -0,0 +1,86 @@
+# pylint: disable=abstract-method,no-self-use
+from typing import Dict
+
+from pytorch_pretrained_bert.modeling import BertConfig, BertModel
+from _pytest.monkeypatch import MonkeyPatch
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.fields import TextField, LabelField
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers import TokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.modules.token_embedders.bert_token_embedder import PretrainedBertModel
+
+
+@DatasetReader.register("bert_classification_test")
+class TwoInstanceReader(DatasetReader):
+    def __init__(self,
+                 lazy: bool = False,
+                 token_indexers: Dict[str, TokenIndexer] = None,
+                 tokenizer: WordTokenizer = None) -> None:
+        super().__init__(lazy)
+        self._token_indexers = token_indexers or {}
+        self.tokenizer = tokenizer or WordTokenizer()
+
+    def _read(self, file_path: str):
+        #            16    2   3    4   3     5     6   8      9    2   14   12  17
+        sentence1 = "[CLS] the quickest quick brown fox jumped over the lazy dog [SEP]"
```

Comment:
I don't think this will work as intended as the word splitter will mangle "[CLS]" here, I think it needs to be added after tokenization.

Reply:
I reverted this change, see above (but I tested it and it didn't mangle, I'm pretty sure the Bert-specific word splitter handles these cases correctly)